activesupport/lib/active_support/core_ext/time/calculations.rb

```diff
@@ -312,4 +312,14 @@ def eql_with_coercion(other)
   end
   alias_method :eql_without_coercion, :eql?
   alias_method :eql?, :eql_with_coercion
+
+  # Returns true if the time is in the past.
+  def past?
```

Comment:
Why this method is being overridden here and not changed in the shared module?

Reply:
Original method is defined in `DateAndTime::Calculations`
https://github.com/rails/rails/blob/40da1c430bc2d8d5cd9a9349993fbdf62e68529f/activesupport/lib/active_support/core_ext/date_and_time/calculations.rb#L44-L46

This module is shared by `Date` and `Time`. I can't change it since we don't have `Date#now` method.